### PR TITLE
Travis-ci

### PR DESCRIPTION
**Pull request by kotsios5**
Add travis.yml, update tests.

working on https://github.com/yclas/yclas/issues/2218

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+git:
+  submodules: false
+
+before_install:
+  - mysql -e 'CREATE DATABASE IF NOT EXISTS openclassifieds;'
+  - phpenv config-rm xdebug.ini
+
+install:
+  - git clone https://github.com/kotsios5/build-travis-ci build   # get required files
```
Comment (neo22s) [inline .travis.yml]: we should have this at our repo, right?

Comment (kotsios5) [inline .travis.yml]: yes, would be better

Comment (neo22s) [inline .travis.yml]: can you transfer it? to https://github.com/yclas/build-travis-ci

Comment (kotsios5) [inline .travis.yml]: I need permissions to create or transfer a repo on https://github.com/yclas..

Comment (neo22s) [inline .travis.yml]: created and invited you ;)

Comment (kotsios5) [inline .travis.yml]: Done!

**Pull request merged**